src/main/scala/org/hammerlab/guacamole/commands/GermlineAssemblyCaller.scala

```diff
@@ -243,11 +262,11 @@ object GermlineAssemblyCaller {
         val probability = math.exp(mostLikelyGenotypeAndProbability._2)
         genotype
           .getNonReferenceAlleles
-          .toSet // Collapse homozygous genotypes
+          .toSet  // Collapse homozygous genotypes
           .filter(_.altBases.nonEmpty)
           .map(allele => {
             CalledAllele(
-              pileup.head.read.sampleName,
+              pileup.elements.head.read.sampleName,
```

Comment:
In many places we use the first element of an array to represent values we assume are the same for all elements in the array; this is generally smelly to me and you get better APIs and code by restructuring things to avoid that, IME. This is a step towards removing `Pileup.head` which is an example of this.